.github/CODEOWNERS

```
@@ -0,0 +1,20 @@
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most precedence.
+
+# Global owners
+* @redhat-developer/rhdh-content @redhat-developer/rhdh-team @josephca
+# Specific areas
+/modules/admin @redhat-developer/rhdh-content @josephca @schultzp2020
+/modules/auth @redhat-developer/rhdh-content @hmanwani-rh @themr0c @josephca @schultzp2020 @PatAKnight
+/modules/authentication @redhat-developer/rhdh-content @hmanwani-rh @themr0c @josephca @schultzp2020
```

```suggestion
/modules/authentication @redhat-developer/rhdh-content @hmanwani-rh @themr0c @josephca @kim-tsao @JessicaJHee @04kash 
```
The security WG is handling a lot of the auth providers today, so let's get them added as codeowners